main.js

```diff
@@ -0,0 +1,64 @@
+let collection = [
+  {
+    title: 'Lorem ipsum',
+    author: 'Testeroo Testyy',
+  },
+  {
+    title: 'Second Book',
+    author: 'Testeroo Testyy',
+  },
+];
+
+function saveLocalStorage() {
+  localStorage.setItem('collection', JSON.stringify(collection));
+}
+
+function loadLocalStorage() {
+  if (JSON.parse(localStorage.getItem('collection'))) {
+    collection = JSON.parse(localStorage.getItem('collection'));
+  }
+}
+
+function addBook(newTitle, newAuthor) {
+  const newBook = {
+    title: newTitle,
+    author: newAuthor,
+  };
+  collection.push(newBook);
+}
+
+function removeBook(removeBook) {
+  collection.pop(Array.prototype.indexOf.call(removeBook.parentElement.children, removeBook));
+}
```

Comment:
- [ ] As the previous reviewer suggested, the delete function is not working as expected. Currently, you don't have a way to guarantee the uniqueness of the books 😱.
 
Here's the problem: Let's say you create two different books with the same title and author and you need to delete the first one. As you're using the `pop` method in this case, it doesn't matter that you click on the first one because `pop` deletes the last element of an array, so it won't work as expected. 🚧 

Please consider working on this, as it is a very important concept! 🤓 🛠️ 

💡 _I suggest you work on this by either adding a unique identifier (Id) property to each book or that you check for uniqueness before creating new books._